lab_1_classify_by_unigrams/main.py

```diff
@@ -37,8 +72,21 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     :param actual: a list of actual values
     :return: the score
     """
-
-
+    if not isistance(predicted,list):
+        return None
+    if not isinstance(actual,list):
+        return None
+    if len(predicted) != len(actual):
+        return None
+    if len(actual)==0:
+        return None
+    else:
+        dlina=len(actual)
+        for index in range(dlina):
+            numeratory += actual[index]
+            numeratorp += predicted[index]
+            numerator += (numeratory - numeratorp)**2
+    mse=(numerator/dlina)
```

Comment:
strange implementation

Reply:
Does it mean wrong and should be changed?